content/modeling/modeling_3.md

```diff
@@ -0,0 +1,101 @@
++++
+title= "Explore Snowplow data"
+weight = 3
+post = ""
++++
+
+Data should now be loaded into your warehouse. In this section we will take a closer look at the output to mitigate data issues and get familiar with the derived tables.
+
+***
+
+#### **Step 1:** Check the output schemas
+Head to the SQL editor of your choice (e.g.: Snowflake Web UI) to check the model's output. You should be able to see three new schemas created:
+1. [your_custom_schema]_***scratch***: drop and recompute models that aid the incremental run
+2. [your_custom_schema]_***derived***: main output models you can use in your downstream models and reporting
+3. [your_custom_schema]_***manifest***: tables that help the integrity and core incremental logic of the model
+
+***
+#### **Step 2:** Run dbt test
+
+Run our recommended selector specified tests to identify potential issues with the data:
+
+```bash
+dbt test --selector snowplow_web_lean_tests
+```
+***
+#### **Step 3:** Explore your data
+
+Take some time to familiarise yourself with the derived tables. You could run a few simple queries such as the ones listed below:
```

Comment:
Think we leave the SQL scripts to be showing always. No other page has this expand / contract for the code, so want to remain consistent. Shame really as I like this expand / contract.

Reply:
After discussing the issue with Jack he proposed another idea, there was a way to limit code blocs to a specific size and have the rest available by scrolling. It works both within and outside the tab so we can keep it consistent. I pushed the code as we both liked it so you can have a look at how it looks.